modules/op-installing-tkn-on-macos.adoc

```diff
@@ -12,11 +12,15 @@ For macOS, the `tkn` CLI is provided as a `tar.gz` archive.
 
 .Procedure
 
-. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.24.0/tkn-macos-amd64.tar.gz[CLI].
+. Download the relevant CLI.
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.24.0/tkn-macos-amd64.tar.gz[macOS]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.24.0/tkn-macos-arm64.tar.gz[macOS on ARM]
 
 . Unpack and unzip the archive.
 
-. Move the `tkn` binary to a directory on your PATH.
+. Move the `tkn` and `tkn-pac` binaries to a directory on your PATH.
```

Comment:
```suggestion
. Move the `tkn` and `tkn-pac` binaries to a directory on your `PATH`.
```
[Contributor's guide](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference): 
Use backticks for all non-GUI "system items", including:
Inline commands, operations, literal values, variables, parameters, settings, flags, environment variables, user input